source/scripting_v3/GTA/Camera/GameplayCamera.cs

```diff
@@ -444,6 +444,53 @@ public static float FirstPersonAimCameraZoomFactor
 		/// </value>
 		public static bool IsFirstPersonAimCamActive => Function.Call<bool>(Hash.IS_FIRST_PERSON_AIM_CAM_ACTIVE);
 
+		/// <summary>
+		/// Gets a value indicating whether a first person vehicle camera is active.
+		/// </summary>
+		/// <value>
+		/// <see langword="true" /> if the <see cref="GameplayCamera"/> is a first person vehicle camera; otherwise, <see langword="false" />.
+		/// </value>
+		public static bool IsFirstPersonVehicleCamActive => Function.Call<bool>(Hash.IS_CINEMATIC_FIRST_PERSON_VEHICLE_INTERIOR_CAM_RENDERING);
+
+		/// <summary>
+		/// Gets a value indicating whether any cinematic camera is active.
+		/// Note that this will also return true if in first person view while inside a vehicle.
+		/// </summary>
+		/// <value>
+		/// <see langword="true" /> if the <see cref="GameplayCamera"/> is a cinematic camera; otherwise, <see langword="false" />.
+		/// </value>
+		public static bool IsAnyCinematicCamActive => Function.Call<bool>(Hash.IS_CINEMATIC_CAM_RENDERING);
+
+		/// <summary>
+		/// Gets a value indicating whether an idle cinematic camera is active.
+		/// </summary>
+		/// <value>
+		/// <see langword="true" /> if the <see cref="GameplayCamera"/> is an idle cinematic camera; otherwise, <see langword="false" />.
+		/// </value>
+		public static bool IsIdleCinematicCamActive => Function.Call<bool>(Hash.IS_CINEMATIC_IDLE_CAM_RENDERING);
+
+		/// <summary>
+		/// Gets a value indicating whether a player-controlled cinematic camera is active.
+		/// Only available in v1.0.1493.0 or later game versions.
+		/// </summary>
+		/// <value>
+		/// <see langword="true" /> if the <see cref="GameplayCamera"/> is a player-controlled cinematic camera; otherwise, <see langword="false" />.
+		/// </value>
+		/// <exception cref="GameVersionNotSupportedException">
+		/// Thrown if called in game versions earlier than v1.0.1493.0.
+		/// </exception>
+		public static bool IsCinematicCamInputActive
+		{
+			get
+			{
+				if (Game.Version < GameVersion.v1_0_1493_0_Steam)
```

Comment:
Nitpick one, but no need to insert if statement here, as `ThrowIfNotSupported` does the same check. Think that method like `ArgumentNullException.ThrowIfNull`, which is introduced in .NET 6.
https://github.com/scripthookvdotnet/scripthookvdotnet/blob/e2326227262696f22be9146cd149461f62d4e5c7/source/scripting_v3/GTA/GameVersionNotSupportedException.cs#L70